lib/src/main/java/com/ioki/textref/TextRef.kt

```diff
@@ -145,19 +145,17 @@ private constructor(
          * @param string The string used to create the TextRef
          * @param args Format args used to format the string
          */
-        fun string(string: String, vararg args: Any): TextRef {
-            return TextRef(string as Any, *args)
-        }
+        fun string(string: String?, vararg args: Any): TextRef =
+            if (string != null) TextRef(string as Any, *args) else EMPTY
```

Comment:
We might as well map empty strings directly to `EMPTY` :)

```suggestion
            if (!string.isNullOrEmpty()) TextRef(string as Any, *args) else EMPTY
```